### PR TITLE
[POAE7-2435] fix col name

### PR DESCRIPTION
**Pull request by Ma-Jian1**
### What changes were proposed in this pull request?
fix bug in col name

### Why are the changes needed?
bug

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
UTs

### Which label does this PR belong to?
BUG

### CODE REVIEW

File: cider/exec/plan/parser/PlanRelVisitor.cpp [hunk 1]
```diff
@@ -431,15 +433,15 @@ void ReadRelVisitor::visit(InputDescContext* input_desc_context) {
   for (int i = 0; i < rel_node_.base_schema().struct_().types_size(); i++) {
     auto type = rel_node_.base_schema().struct_().types(i);
     auto sti = getSQLTypeInfo(type);
-    ColumnInfoPtr colum_info_ptr =
-        std::make_shared<ColumnInfo>(100, cur_table_id, i, "col_" + i, sti, false);
+    ColumnInfoPtr colum_info_ptr = std::make_shared<ColumnInfo>(
+        fake_db_id, cur_table_id, i, fmt::format("col_{}", i), sti, false);
```
Comment (yma11) [inline cider/exec/plan/parser/PlanRelVisitor.cpp:437]: When will `"col_" + i` not work and how does it looks like?

Comment (Ma-Jian1) [reply cider/exec/plan/parser/PlanRelVisitor.cpp:437]: `"col_" + i` means `"col_"[i]`

Comment (yma11) [reply cider/exec/plan/parser/PlanRelVisitor.cpp]: Verified. Yes we should wrap this `i` with `std::to_string.

File: cider/exec/plan/parser/PlanRelVisitor.cpp [hunk 2]
```diff
@@ -24,6 +24,8 @@
  * @brief   Plan Rel visitor
  **/
 
+#include <fmt/core.h>
```
Comment (yma11) [inline cider/exec/plan/parser/PlanRelVisitor.cpp:27]: remove this?

Comment (Ma-Jian1) [reply cider/exec/plan/parser/PlanRelVisitor.cpp:27]: done

**Review by bigPYJ1151 — DISMISSED** [review]

**Review by bigPYJ1151 — APPROVED** [review]

**Pull request merged**